packages/client/hmi-client/src/views/DocumentView.vue

```diff
@@ -0,0 +1,121 @@
+<script setup lang="ts">
+import Document from '@/components/articles/Document.vue';
+import TabContainer from '@/components/tabs/TabContainer.vue';
+import { ref, watch, computed } from 'vue';
+import { Tab } from '@/types/common';
+import useResourcesStore from '@/stores/resources';
+import { Project } from '@/types/Project';
+import { RouteName } from '@/router/routes';
+import { useTabStore } from '@/stores/tabs';
+import { isEmpty } from 'lodash';
+import ResourcesList from '@/components/resources/resources-list.vue';
+
+const props = defineProps<{
+	assetId?: string;
+	project: Project;
+}>();
+
+const emit = defineEmits(['show-data-explorer']);
```

Comment:
With my recent changes, data explorer is a view, so we can route to it instead of using emits.

Reply:
Done